src/test/java/io/anserini/search/query/BagOfWordsQueryGeneratorTest.java

```diff
@@ -29,7 +30,7 @@
 public class BagOfWordsQueryGeneratorTest {
   @Test
   public void test1() {
-    Analyzer analyzer = new EnglishAnalyzer();
+    Analyzer analyzer = IndexCollection.DEFAULT_ANALYZER;
```

Comment:
What will be the default analyzer? Just curious, why don't we use the English analyzer?

Reply:
This is a recent change to standardize the default analyzer... basically, we _are_ using the `EnglishAnalyzer()`

Reply:
Oh ok, thanks!